packages/headless/src/controllers/facets/facet-search/category/headless-category-facet-search.ts

```diff
@@ -24,6 +24,7 @@ export function buildCategoryFacetSearch(
   const {dispatch} = engine;
   const {options} = props;
   const {facetId} = options;
+  const numberOfValues = options.numberOfValues || 10;
```

Comment:
Is the search default number of values the same as the facet's default number of value, and if there is, shouldn't there already be a default value for that option?

Reply:
Yes, there is a variable `defaultFacetSearchOptions` containing the defaults that is exported from `facet-search-reducer-helpers.ts`. We should not hard-code the default.


We can also limit the variable's scope to the `select` function, since it is not used anywhere else.

Reply:
Yeah, the default options wasn't being used here so typescript complained numberOfValues may potentially be undefined. Fixed this.